fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/KeyExpressionExpansionVisitor.java

```diff
@@ -121,16 +122,18 @@ public GraphExpansion visitExpression(@Nonnull FieldKeyExpression fieldKeyExpres
                 .add(fieldName)
                 .build();
         final Value value;
+        final Column<?> column;
```

Comment:
This updates the `Column` from an unnamed column to one that preserves the field name. This mainly makes it easier to track where the field comes from, though it's not strictly necessary for this PR.

Reply:
Yeah, I can do that as a precaution